src/pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,207 @@
+            onChange={e =>
+              this.setState({
+                ...this.state,
+                user: { ...this.state.user, username: e.target.value },
+              })
+            }
+          />
+        </div>
+        <div className="input">
+          <div className="label">Email</div>
+          <input
+            type="email"
+            name="email"
+            required
+            onChange={e =>
+              this.setState({
+                ...this.state,
+                user: { ...this.state.user, email: e.target.value },
+              })
+            }
+          />
+        </div>
+        <div className="input">
+          <div className="label">Password</div>
+          <input
+            type="password"
+            name="password"
+            required
+            minLength="8"
+            placeholder="Upercase letters and numbers"
+            onChange={e =>
+              this.setState({
+                ...this.state,
+                user: { ...this.state.user, password: e.target.value },
+              })
+            }
+          />
+        </div>
+        <div
+          className="button"
+          onClick={() =>
+            this.form.current.reportValidity() ? this.onSubmit() : null
+          }
+        >
+          {loading ? this.spinner() : null}
+          <span>REGISTER</span>
+        </div>
+        <div className="login">
+          Already have an account ?
+          <Link to="/login">
+            <b> Login</b>
+          </Link>
+        </div>
+      </form>
+    );
+  }
+
+  render() {
+    return (
+      <section className="signup container d-flex justify-content-center align-items-center col-md-8 col-sm-12 ">
+        <div className="row col-md-12 col-sm-12 layout">
+          <div className="personal h-100 col-md-6 col-sm-12">
+            <div>
+              <div className="title">Sign up</div>
+              <div className="subtitle">Register a personal account</div>
+              <div className="social">
+                <div>
+                  <img src={google} alt="google" />
+                </div>
+                <div>
+                  <img src={facebook} alt="facebook" />
+                </div>
+                <div>
+                  <img src={github} alt="github" />
+                </div>
+              </div>
+            </div>
+            <div className="girl">
+              <img src={girl} alt="girl" />
+            </div>
+          </div>
+          {this.state.toggled ? this.organizationForm() : this.organization()}
+        </div>
+      </section>
+    );
+  }
+}
+
+/**
+ * Maps the state to props
+ * @param {*} { auth }
+ * @returns {object} props
+ */
+export const mapStateToProps = ({ auth }) => {
+  const { loading, error } = auth;
+  return {
+    loading,
+    error,
+  };
+};
+
+/**
+ * Maps dispatches to props
+ * @param {*} dispatch
+ * @returns {object} props
+ */
+export const mapDispatchToProps = dispatch => {
```

Comment:
@gracelungu Try to shorten the returned object without using the `return` keyword